eox_nelp/processors/xapi/event_transformers/initialized_events.py

```diff
@@ -22,26 +22,10 @@ class InitializedCourseTransformer(XApiTransformer):
         display=LanguageMap({constants.EN: constants.INITIALIZED}),
     )
 
-    def get_object(self):
+    @cached_property
```

Comment:
This is only cached in memory with the life of the instance?.
So would be cached only for a trigger of  @XApiTransformersRegistry.register("nelc.eox_nelp.initialized.course") register event? 
Why do you want it to be cached?

Reply:
https://docs.djangoproject.com/en/5.0/ref/utils/#django.utils.functional.cached_property
yes, yes, performance